admissions/admissions/templatetags/vite.py

```diff
@@ -1,20 +0,0 @@
-from django import template
-from django.conf import settings
-from django.utils.safestring import mark_safe
-
-register = template.Library()
-
-
-@register.simple_tag
-@mark_safe
-def vite_react_preamble():
-    if settings.DJANGO_VITE_DEV_MODE:
-        return f"""<script type="module">
-          import RefreshRuntime from 'http://localhost:{settings.DJANGO_VITE_DEV_SERVER_PORT}/@react-refresh'
```

Comment:
You should probably remove these settings objects now as well.

Reply:
Or maybe some of these are needed? Seems like django-vite exptects the `DEV_SERVER_PORT`?